getify/LABjs
# Add error handler support to LABjs

I also added a `Makefile` since the build instructions aren't super helpful.

Test plan:
 - `make`
 - copy into webapp
 - See that it catches errors

@jeresig
